examples/gke-security-best-practices/builder/platform-uploads/tmp/roles/cis/README.md

```diff
@@ -0,0 +1,42 @@
+# Ansible role for hardening Ubuntu 14.04
```

Comment:
14.04 is fairly old and reaches EOL April 2019, any reason we wouldn't use 16.04?

Reply:
this is an artifact. I will address. the packer actually produces a 16.04 image

Reply:
removed in favor of a pre-build CIS security hardened image image from the GCP marketplace